homeworkLecturesW11D05/src/ba/bitcamp/homeworkLecturesW11D5/BitResponse.java

```diff
@@ -0,0 +1,60 @@
+package ba.bitcamp.homeworkLecturesW11D5;
+
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+/**
+ * 
+ * @author Gordan
+ * 
+ *         This class represents one message that server reads from file and
```

Comment:
You should not document where the class is used but rather what is the purpose of the class. Imagine a how would 'String' java-doc look like if its creators wanted to document its usages.